package.json

```diff
@@ -34,6 +34,7 @@
     "jest:e2e": "jest -c jest.config.e2e.js --",
     "coverage": "cross-env NODE_ENV=test LOG_LEVEL=silent MOCK_REDIS=1 FEED_URL_INTERVAL_MS=200 jest -c jest.config.js --collectCoverage --",
     "jest-watch": "cross-env MOCK_REDIS=1 jest --watch --",
+    "jest-update": "cross-env MOCK_REDIS=1 jest --u",
```

Comment:
Include `-c jest.config.js` and also pass `--` at the end, in case you only want to update some subset of tests